screens/PreviewDirections.js

```diff
@@ -42,24 +42,72 @@ const getFilteredDetailedInstructions = (jsonLeg) => {
         },
         steps: []
     };
-    directionObject.steps = jsonLeg.steps.map(step => {
-
-        return {
-            distance: step.distance.text,
-            duration: step.duration.text,
-            polylines: decodedPolylinesAlgo(step.polyline.points),
-            startLocation: {
-                latitude: step.start_location.lat,
-                longitude: step.end_location.lng
-            },
-            endLocation: {
-                latitude: step.end_location.lat,
-                longitude: step.end_location.lng
-            },
-            htmlInstructions: instructionsHtmlStyle + step.html_instructions + "</div>",
-            travelMode: step.travel_mode
-        };
-    });
+    if (transportType != "transit") {
+        directionObject.steps = jsonLeg.steps.map(step => {
+            return {
+                distance: step.distance.text,
+                duration: step.duration.text,
+                polylines: {
+                    values: decodedPolylinesAlgo(step.polyline.points),
+                    color: (step.travel_mode == "DRIVING" | step.travel_mode == "BICYCLING") ? "blue" : "black"
```

Comment:
<img width="747" alt="Screen Shot 2020-04-05 at 5 48 44 PM" src="https://user-images.githubusercontent.com/35850933/78510763-b3ca4000-7765-11ea-8ca8-efe9fffe7de4.png">

Reply:
If this is not an actual issue let me know and I will resolve it as "wont fix"

Reply:
nice one!